lib/danger/ci_source/circle_api.rb

```diff
@@ -16,7 +16,7 @@ def pull_request_url(env)
 
       if url.nil? && !env["CIRCLE_PROJECT_USERNAME"].nil? && !env["CIRCLE_PROJECT_REPONAME"].nil?
         repo_slug = env["CIRCLE_PROJECT_USERNAME"] + "/" + env["CIRCLE_PROJECT_REPONAME"]
-        token = env["DANGER_CIRCLE_CI_API_TOKEN"] || env["CIRCLE_CI_API_TOKEN"]
+        token = env["DANGER_CIRCLE_CI_API_TOKEN"]
```

Comment:
I unified these variables. The docs only mention `DANGER_CIRCLE_CI_API_TOKEN`, so I'm guessing the other one is there for compatibility. I hope it's fine to remove support for that.. Older versions of Danger have CircleCI API failures anyway, so I don't think this will unexpectedly break anyone's config.

Reply:
Yeah, I likely kept it around for backwards compatibility for myself at some point